reporterdashboard/example-reports/r1.json

```diff
@@ -0,0 +1,78 @@
+{
+    "results": {
```

Comment:
Does python have some kind of (de)serialize library? Would be neat to have typings on these json files, otherwise it can be a source of bugs

Reply:
`json.loads()` and `json.dumps()` (and equivalents) are already (de)serialize methods. The Python JSON decoder already has enough information to correctly load different types (e.g. string vs. int vs. float) (see also [the docs](https://docs.python.org/3/library/json.html#json-to-py-table)). It is the job of the plugin that generates the report to validate that it actually outputs the correct values (`1234` (correct) vs. `"1234"` (incorrect), when trying to output a number).

Reply:
That may be done somewhere in the future, but for now I would say the documentation ([link](https://github.com/aron-hoogeveen/energy-consumption-reporter/blob/main/docs/report-file-format.md)) of the JSON format is good enough for people not having to guess how the shape of the format will look like

Reply:
Fair enough, was just wondering if there was some python way to do it